build.gradle

```diff
@@ -48,6 +48,13 @@ task ktlint(type: JavaExec, group: LifecycleBasePlugin.VERIFICATION_GROUP) {
   args '**/src/**/*.kt', '--baseline=ktlint-baseline.xml', '--verbose'
 }
 
+task ktlint_experimental(type: JavaExec, group: LifecycleBasePlugin.VERIFICATION_GROUP) {
+  description = "Check Kotlin code style with experimental rules."
+  classpath = configurations.ktlint
+  main = 'com.pinterest.ktlint.Main'
+  args '**/src/**/*.kt', '--verbose', '--experimental'
```

Comment:
I think we don't need a new task for that, I'd just add `--experimental` to the task above. Otherwise, we'd also need to update the CI workflow

Reply:
Yes indeed. I was not sure whether you was willing to integrate it in the default pipeline. But I definitely am in favor of this.